mbq/atomiq/producers.py

```diff
@@ -49,8 +49,36 @@ def _is_running_within_transaction(self):
         save points and expect there at least to be 2: 1 from TestCase and 1 user-defined.
         """
         if RUNNING_TESTS:
-            if len(db_connection.savepoint_ids) < 2:
-                return False
+            import inspect
+            from django.test import TestCase
+            inSetup = False
```

Comment:
nit: why does all this code have camel case variable names?

Reply:
Solid nit

Reply:
I actually have no idea, that's just how the code wrote itself.